nilearn/_utils/ndimage.py

```diff
@@ -47,3 +47,74 @@ def get_border_data(data, border_size):
         data[:, :, :border_size].ravel(),
         data[:, :, -border_size:].ravel(),
     ])
+
+
+def _peak_local_max(image, min_distance=10, threshold_abs=0, threshold_rel=0.1,
+                    num_peaks=np.inf):
+    """Find peaks in an image, and return them as coordinates or a boolean array.
+
+    Peaks are the local maxima in a region of `2 * min_distance + 1`
+    (i.e. peaks are separated by at least `min_distance`).
+
+    NOTE: If peaks are flat (i.e. multiple adjacent pixels have identical
+    intensities), the coordinates of all such pixels are returned.
+
+    Parameters
+    ----------
+    image : ndarray of floats
```

Comment:
the variable name ``image`` typically refers to a nifti-like object, not to an array. Perhaps consider shifting to ``image_data``or something like that to increase code consistency.